GenderPayGap.WebUI/Views/Viewing/Finder/Parts/Facets/OptionSelect.cshtml

```diff
@@ -16,7 +16,7 @@
                 @foreach (OptionSelect item in Model.Metadata)
                 {
                     <label for="@(item.Id)" @(item.Disabled ? "disabled" : string.Empty)>
-                        <input name="@(Model.Group)" @(item.Disabled ? "disabled" : string.Empty) value="@(item.Value)" id="@(item.Id)" type="checkbox" @(queryLookup.Contains(item.Value) ? "checked" : "") aria-label="@(item.Id)"/>
+                        <input name="@(Model.Group)" @(item.Disabled ? "disabled" : string.Empty) value="@(item.Value)" id="@(item.Id)" type="checkbox" @(queryLookup.Contains(item.Value) ? "checked" : "") aria-label="@(item.Id != "Size0" ? item.Label + " employees" : "Number of employees not provided")"/>

```

Comment:
I think this OptionSelect.cshtml is used for all the filters (not just number of employees)
So, `item.Label + "employees"` might not always be relevant.

Note: the `item.Label` is there below (on line 20) and I think the aria-label overrides this, so it might just be a case of removing the `aria-label`

Reply:
Oh yeah, whoops! 🤦